### PR TITLE
Add interfaces for spreed signaling messages

### PR DESCRIPTION
**Pull request by asinner**

### CODE REVIEW

**Review by niuyunsi — APPROVED** [review]

File: src/data/SpreedSignaling.ts [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+// This file does not cover the following message types:
+// - Screenshare
+// - YouTubeVideo
+// - Presentation
+// - Chat
+
+// NOTE(andrews): SpreedMessage represents any message sent from the Spreed WSS.
+// The Data member contains the type of event as well as the relevant payload.
+export interface SpreedMessage {
+    Data: SpreedMessageData;
```
Comment (linweigao) [inline src/data/SpreedSignaling.ts:14]: Just want to confirm that Spreed contract is upper case.

File: src/data/SpreedSignaling.ts [hunk 2]
```diff
@@ -0,0 +1,170 @@
+// NOTE(andrews): This file contains most of the messages that
```
Comment (linweigao) [inline src/data/SpreedSignaling.ts:1]: What's the plan for supporting other webRTC service?

Should we have a more general interface name?

Comment (asinner) [reply src/data/SpreedSignaling.ts:1]: I'm not sure, I'm still brainstorming the abstraction. This file is only meant to add type information to Spreed-specific signaling events.

File: src/data/SpreedSignaling.ts (hunk 1 above)
Comment (asinner) [reply src/data/SpreedSignaling.ts:14]: Yes, they use uppercase for (almost) every contract

**Review by linweigao — APPROVED** [review]

**Pull request merged**